### ISSUE TITLE
Report Metric Development relies on datTest which is created in CODE/31

### ISSUE THREAD

**Issue by stephlocke** [Chicago/food-inspections-evaluation#99]
The [report](https://github.com/Chicago/food-inspections-evaluation/blob/master/REPORTS/Metric_Development.Rmd) only sources the 30 file, but datTest is created in 31. As a result the report should be updated to source both files to ensure it works in a standalone fashion:
```
source("CODE/30_glmnet_model.R", echo = FALSE)
source("CODE/31_glmnet_model_evaluation.R", echo = FALSE)
```

**Comment by stephlocke** [comment]
Looks like PR #94 did not involve updating the reports ... a number of the others also expect datTest and glm_pred

**Comment by geneorama** [comment]
@stephlocke Yes.  Those updates for in PR #94 were in response to something else, and I never got a chance to go back and fix the inputs for the report (which was a work in progress anyway).  As such the path from the data to the report is currently broken.

**Comment by geneorama** [comment]
@stephlocke Granted, this isn't a very satisfying solution, but you should be able to checkout commit 150e399782714e4f597e07956eb3161a2b389628, and that should allow you to compile the document. So, at the command line `git checkout 150e399`.  Nothing's changed *yet* in the model, so you'd have the same numbers.
